--- Android.bp.orig	2025-01-30 14:36:16.055717515 +0000
+++ Android.bp	2025-01-30 14:36:20.323962732 +0000
@@ -2,17 +2,10 @@
 // Do not modify this file because the changes will be overridden on upgrade.
 
 package {
-    default_applicable_licenses: ["external_rust_crates_flatbuffers_license"],
+    default_applicable_licenses: ["external_flatbuffers_license"],
     default_team: "trendy_team_android_rust",
 }
 
-license {
-    name: "external_rust_crates_flatbuffers_license",
-    visibility: [":__subpackages__"],
-    license_kinds: ["SPDX-license-identifier-Apache-2.0"],
-    license_text: ["LICENSE"],
-}
-
 rust_library {
     name: "libflatbuffers",
     host_supported: true,
